File: test/lineTest.R
require(ggplot);

let x = seq(-5,5, by = 0.2);
let y = sin(x);
let line_data = data.frame(x, y);

print(line_data, max.print = 13);

let plot_line = function() {
    ggplot(line_data, aes(x = "x", y = "y"), padding = "padding: 200px 500px 200px 200px;")
    + geom_line(width = 8, show.legend = TRUE, color = "Jet")
    ;
}

let do_plot = function() {
    let w = 2400;
    let h = 1600;

    bitmap(file = file.path(@dir,  "line_sin.png"), width = w, height = h) {
        plot_line();
    }

    svg(file = file.path (@dir,  "line_sin.svg"), width = w, height = h) {
        plot_line();
    }
}

do_plot();

File: test/lineTest.py
import ggplot

x = seq(-5, 5, by = 0.2)
y = sin(x)
input = data.frame(x = x, y = y)

def plotfile(filepath):
   print("previews of the plot table:")
   print(input, max.print = 13)
   
   plt = ggplot(input, aes(x = "x", y = "y"), padding = "padding: 200px 500px 200px 200px;", width = 2400, height = 1600) 
   plt = plt + geom_line(width = 8, show.legend = TRUE, color = "Jet")

   bitmap(plt, file = filepath)
   
   
plotfile(`${@dir}/line_sin_py.png`)